hypershift-operator/controllers/platform/aws/controller.go

```diff
@@ -523,43 +523,55 @@ func (r *AWSEndpointServiceReconciler) delete(ctx context.Context, awsEndpointSe
 		ServiceIds: []*string{aws.String(serviceID)},
 	})
 	if err != nil {
-		log.Error(err, "Failed to delete vpc endpoint service configuration", "serviceID", serviceID)
-		existingConnectionsResult, describeConnectionsErr := r.ec2Client.DescribeVpcEndpointConnectionsWithContext(ctx, &ec2.DescribeVpcEndpointConnectionsInput{
-			Filters: []*ec2.Filter{
-				{
-					Name:   aws.String("service-id"),
-					Values: []*string{aws.String(serviceID)},
-				},
-			},
-		})
-		if describeConnectionsErr != nil {
-			log.Error(describeConnectionsErr, "Failed to get existing connections", "serviceID", serviceID)
-			return false, unwrapError(describeConnectionsErr)
-		}
-		var existingEndpointIDs []*string
-		for _, conn := range existingConnectionsResult.VpcEndpointConnections {
-			state := aws.StringValue(conn.VpcEndpointState)
-			switch state {
-			case "pendingAcceptance", "pending", "available":
-				existingEndpointIDs = append(existingEndpointIDs, conn.VpcEndpointId)
-			}
-		}
-		if len(existingEndpointIDs) > 0 {
-			if _, rejectEndpointsErr := r.ec2Client.RejectVpcEndpointConnectionsWithContext(ctx, &ec2.RejectVpcEndpointConnectionsInput{
-				ServiceId:      aws.String(serviceID),
-				VpcEndpointIds: existingEndpointIDs,
-			}); rejectEndpointsErr != nil {
-				return false, unwrapError(rejectEndpointsErr)
-			}
-		}
+		// Only client error codes are returned here
+		// https://docs.aws.amazon.com/AWSEC2/latest/APIReference/errors-overview.html#CommonErrors
```

Comment:
Which point is this comment making? As far as I can see in the link, `ExistingVpcEndpointConnections` and `InvalidVpcEndpointService.NotFound` are also client error codes, so I would think we will always return here if `InvalidVpcEndpointService.NotFound`.
I know we won't because the comment below
`DeleteVpcEndpointServiceConfigurations doesn't return...`

Reply:
Oh, yeah I meant this comment to make the point that "Only common client error codes are returned here" - but I left out "common" and also agree that this comment is redundant if I have the comment below. I'll remove this one